MDAnalysisData/tests/test_base.py

```diff
@@ -83,7 +83,40 @@ def remote_topology():
     # small-ish file
     return adk_equilibrium.ARCHIVE['topology']
 
+@pytest.mark.online
```

Comment:
Perhaps leave this included (even though it's "online") – but it runs fast and does not slow down the tests. It would make me feel better if I knew that at least one real remote fetch is done whenever the test suite runs.